### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by mohammadsalem** [waqarmuhammad1/ICARDA#1]
Hi @waqarmuhammad1,
I was trying to install the project inside a docker (ubuntu:xenial image), I was able to run both servers on "localhost/Predator" and "localhost:8091", but I was not able to login (the login request returns 500).
`sudo apt-get install libcouchbase-dev build-essential python36-dev python36-pip` this line return packages not found for "build-essential" and "python36-dev python36-pip". In the link for "couchbase python dependency" the command is different `sudo apt-get install libcouchbase-dev build-essential python-dev python-pip` and it worked. Then I got some errors while setting-up python environment, I fixed the errors and completed it, when I run `python3 RunAPI.py` I got an error that something is missing for mysql.

Another point is the security, if we are going to use it with MEL, it will have access to the database and the uploaded files, can you confirm that it has no vulnerabilities?

**Comment by waqarmuhammad1** [comment]
Dear Salem,

You have to install mysql driver and python api aswell, i have used flask rest api which provide some security, file extension and type checks are done in the script there isn't any additional security in place, but i believe if someone will try to upload any kind of bash script or something like that my scripts won't execute such files.

Waqar

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by mohammadsalem** [comment]
@waqarmuhammad1 Thanks, I had another python version and that what caused the problem.
- I was able to install and run it inside a docker container #2.
- When uploading file for image detection, I got this response `Cannot interpret feed_dict key as Tensor: Tensor Tensor("Placeholder:0", shape=(7, 7, 3, 64), dtype=float32) is not an element of this graph.`, any ideas how to solve it? 
- Also the request took 20 seconds, is it normal?

**Comment by waqarmuhammad1** [comment]
Yes, it initializes the whole model on each request, i wasn't able to use the same previously initialized model again, thats why its taking this much time, i am looking into it and as soon as i figure it out i will update the code
